ang/civicase/ActivityCard--Big.html

```diff
@@ -108,13 +81,13 @@
         </div>
         <!-- End - With -->
         <!-- Big Date -->
-        <div ng-if="activity.category.indexOf('milestone') > -1" class="civicase__highlighted-date" ng-class="{'civicase__highlighted-date--overdue': activity.is_overdue}">
+<!--         <div ng-if="activity.category.indexOf('milestone') > -1" class="civicase__highlighted-date" ng-class="{'civicase__highlighted-date--overdue': activity.is_overdue}">
           <div class="civicase__highlighted-date__inner">
             <span class="civicase__highlighted-date__due-on"> Due on </span>
             <span class="civicase__highlighted-date__ddmm"> {{formatDate(activity.activity_date_time, 'DD MMM') }} </span>
             <span class="civicase__highlighted-date__yyyy"> {{formatDate(activity.activity_date_time, 'YYYY') }} </span>
           </div>
-        </div>
+        </div> -->
         <!-- End Big Date -->
       </div>
       <!-- Activity Body -->
```

Comment:
Line 98 and 99 can be combined
`<div class="civicase__activity-no-result-icon" ng-class="{'civicase__activity-no-result-icon--milestone': activity.category.indexOf('milestone') > -1>, 'civicase__activity-no-result-icon--activity': activity.category.indexOf('milestone') === 0}"></div>`

same for line 100 and 101
same for line 102 and 103
same for line 104-113 and 114-123